core/src/test/scala/org/apache/spark/storage/BlockManagerDecommissionUnitSuite.scala

```diff
@@ -38,6 +38,9 @@ class BlockManagerDecommissionUnitSuite extends SparkFunSuite with Matchers {
   private val sparkConf = new SparkConf(false)
     .set(config.STORAGE_DECOMMISSION_SHUFFLE_BLOCKS_ENABLED, true)
     .set(config.STORAGE_DECOMMISSION_RDD_BLOCKS_ENABLED, true)
+    // Just replicate blocks as fast as we can during testing, there isn't another
+    // workload we need to worry about.
+    .set(config.STORAGE_DECOMMISSION_REPLICATION_REATTEMPT_INTERVAL, 10L)
```

Comment:
Should this also be 10 seconds ? 

"as fast as we can" does not usually jibe well with "interval of 10 seconds" :-P.

Reply:
Well compared the the default interval. What about if we say "quickly"?

Reply:
This code is also replicated in BlockManagerDecommissionIntegrationSuite ... so I think that should be changed as well.

Reply:
Done :)